content/docs/azure/azureEndtoEnd.md

```diff
@@ -0,0 +1,171 @@
+
+```export VERSION_TAG=1```
+
+Each docker image will be built and uploaded to the cloud using the Container Registry. 
+
+Note: If you would like to test a container locally, you can use the `docker run -it ${REGISTRY_PATH}<CONTAINER NAME>:$(VERSION_TAG}` before pushing to Container Registry. 
+	//Starting in the 'code' directory of azurepipeline
+
+	cd preprocess
+	docker build . -t ${REGISTRY_PATH}/preprocess:${VERSION_TAG}
+	docker push ${REGISTRY_PATH}/preprocess:${VERSION_TAG}
+
+	cd ../training
+	docker build . -t ${REGISTRY_PATH}/training:${VERSION_TAG}
+	docker push ${REGISTRY_PATH}/training:${VERSION_TAG}
+
+	cd ../register
+	docker build . -t ${REGISTRY_PATH}/register:${VERSION_TAG}
+	docker push ${REGISTRY_PATH}/register:${VERSION_TAG}
+
+	cd ../profile
+	docker build . -t ${REGISTRY_PATH}/profile:${VERSION_TAG}
+	docker push ${REGISTRY_PATH}/profile:${VERSION_TAG}
+
+	cd ../deploy
+	docker build . -t ${REGISTRY_PATH}/deploy:${VERSION_TAG}
+	docker push ${REGISTRY_PATH}/deploy:${VERSION_TAG}
+
+Once all of the images are pushed successfully, modify the pipeline.py file to use the appropriate image for each pipeline step.
+
+## Running and deploying your pipeline
+### Compile
+To compile the pipeline, simply open a terminal and navigate to the azurepipeline/code folder. Run the following command to generate a pipeline in the tar.gz format:		
+		`python pipeline.py`
+### Run & Deploy
+Upload the pipeline.tar.gz file to the pipelines dashboard on your Kubeflow deployment.
+
+![Upload pipeline](images/pipelinedash.PNG)
+ 
+ Create an experiment and then create a run using the pipeline you just uploaded. 
+
+ ![Pipeline inputs](images/pipelinesInput.PNG)
+
+ The finished pipeline should have five completed steps.
+
+ ![Completed pipeline](images/finishedRunning.PNG)
+
+### Pushing images for scoring
+Once your pipeline has finished successfully, you can visit the Azure portal to find your deployment url. Go to your ML workspace dashboard and select “Deployments” from the sidebar. Click on the most recent deployment. You should see a link under “Scoring URI”. You can whatever method you know best to send a GET/POST request with an image of a taco or a burrito to this url and it should return whether or not the image is of a taco or a burrito. 
+
+The easiest method is to find a url of an image of a taco or a burrito and append it to your scoring url as follows: `<scoring_url>?image=<image_url>`
+
+![Final output](images/finalOutput.PNG)
+
+## Clean Up Your Azure Environment
+When you are done, make sure you delete your resource group to avoid extra charges.
+
+	az group delete -n MyResourceGroup
+You can optionally choose to delete individual resources on your clusters using the [Azure cluster docs](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-tutorial-delete-cluster).
+
+## Next Steps
+Build your own pipeline using the [Kubeflow Pipelines SDK](https://www.kubeflow.org/docs/pipelines/sdk/sdk-overview/).
```

Comment:
Use relative links for all docs within the kubeflow.org site. This helps prevent people hopping involuntarily to the prod site when reviewing your doc, and makes the docs easily movable - especially when we branch the docs to a new version.